framework/lib/components/date-picker/components/date-field/trigger.tsx

```diff
@@ -14,7 +14,7 @@ export const Trigger = (props: TriggerProps) => {
     <Button
       { ...buttonProps }
       ref={ ref }
-      size="sm"
+      size="md"
```

Comment:
Change this back to sm, otherwise the bg on hover stretches out weirdly
![Screenshot 2023-10-19 at 13 41 28](https://github.com/mediatool/northlight/assets/90923099/fa068d78-ff79-4101-8b2f-2fb2a7ada930)

Reply:
well , if it's sm then it's not accessible sometimes. It's needs to look good in sm/md/lg etc.. I'm aware the background stretches a bit but this is a quickfix , we may need to give it a more attention next time, i just wanted to quickfix the bug so the users won't struggle with the overlapping elements.